users/klackygears/klackygears.h

```diff
@@ -0,0 +1,92 @@
+#ifndef USERSPACE
+#define USERSPACE
+
+#include "quantum.h"
+#include "wrappers.h"
+
+#pragma once
+#include "quantum.h"
+#include "version.h"
+#include "eeprom.h"
+
+#ifdef TAP_DANCE_ENABLE
+  #include "tap_dances.h"
+#endif // TAP_DANCE_ENABLE
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+
+#endif
```

Comment:
This isn't needed anymore, as there are functions to query all of the config aspects now.

```suggestion
```